app/design/frontend/Magento/theme/Magento_Cms/web/css/source/_module.less

```diff
@@ -0,0 +1,116 @@
+//  Common
+//  _____________________________________________
+
+& when (@media-common = true) {
+
+    .cms-home.cms-index-index {
+        .page-wrapper > main.page-main {
+            max-width: 1920px;
+            padding-left: 0;
+            padding-right: 0;
+        }
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    p {
+        margin-bottom: 0;
+        margin-top: 0;
+    }
+
+    .list {
+        list-style: none;
+    }
+
+    .reset {
+        margin-top: 0;
+        margin-bottom: 0;
+        padding: 0;
+    }
+
+    .baner-wrapper {
+        background-color: #f3f3f3;
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
```

Comment:
please check flex mixins and update:
code/vendor/magento/magento2-base/lib/web/css/source/lib/_utilities.less
from line 412